examples/svm.py

```diff
@@ -110,3 +110,10 @@ def svm_from_cfg(cfg):
 inc_value = svm_from_cfg(incumbent)
 
 print("Optimized Value: %.2f" % (inc_value))
+
+
+# We can also validate our results (though this makes a lot more sense with instances)
```

Comment:
So, to see if I get this right:
Let's say we have instances 1-50 for training and 60-100 for test.
To evaluate one training instance (for example 3), I train the model on the remaining training-instances (that would be 1, 2, 4 - 50)? And, in turn, if I evaluate a test-instance, I train on remaining test-instances? This seems very computationally expensive to me, training a SVM for every instance. What do you think about (hardcoding) packages, so we have instances `1-to-10`, `11-to-20`, ... ? This would make all learning a bit more general.